substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/SystemPropertiesSupport.java

```diff
@@ -103,10 +103,12 @@ protected SystemPropertiesSupport() {
         }
 
         initializeProperty("java.vm.name", "Substrate VM");
-        initializeProperty("java.vm.vendor", "Oracle Corporation");
+        initializeProperty("java.runtime.name", "Substrate VM");
```

Comment:
The `runtime` property has nothing to do with Substrate VM as far as I know, because "runtime" means JDK (in contrast to "VM").